0006/README.md

```diff
@@ -0,0 +1,31 @@
+---
+id: 0009
+title: Automated GitHub Action Runner
+status: ideation 
+authors: David McKay <david@rawkode.com>
+---
+
+## Summary:
+
+The `tink` and `portal` repositories use GitHub Actions to run their automated tests on pull-request and push to master.
+
+A dedicated GitHub Action runner is provided for the [Tinkerbell organization](https://github.com/tinkerbell), this runner was manually created and provisioned by [Gianluca](https://github.com/gianarb); this leaves the runner vulnerable; should it suffering any problems, restoring it will be a manual process.
+
+
+## Goals:
+
+- Automate the creation and provisioning of the dedicated runner
+- Convert all Tinkerbell repositories to using GitHub Actions with our dedicated runner
+
+### Future Goals
+
+We may wish to encourage all Tinkerbell repositories to utilise the dedicated runner and GitHub Actions, for consistency across the organization. Currently, all other repositories use [Drone](https://drone.io).
```

Comment:
This is a non goal imho. We should try to avoid any connection between Tinkerbell and Packet